CMSgov/design-system
# [NO-TICKET] Remove old maturity model status and page

## Summary

- Remove maturity model page
- Remove status: draft from pages 

## How to test

1. Run `yarn start` and visit pages for accordion, filter chip, pagination, table, tooltip, and USA banner to see there is no longer a draft badge in the header
2. See that there is no longer a mtuirt page under the guidelines section in the left nav

## Checklist




### If this is a change to code:


### If this is a change to documentation:
